initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -468,6 +468,20 @@ startCmd() {
         fi
     fi
 
+    # apply user limits we explicitly added ...
+    myid=$(id -u 2>/dev/null)
+    if [[ "${myid}" = "0" ]] ; then
+        nofile_user=$(su - ${user} -c 'bash -c "ulimit -Hn"' 2>/dev/null)
+        rc=$?
+        if [[ $rc -ne 0 ]] ; then
+            nofile_user=$(grep 'hpcc* hard* nofile' /etc/security/limits.conf 2>/dev/null | awk '{print $4}')
```

Comment:
NOTE: what if hpcc user limits where instead in /etc/security/limits.d/* file(s) ?

Reply:
Yeah I think we should check for the limits.d file as well,